src/m365/spo/commands/web/web-set.ts

```diff
@@ -53,6 +54,7 @@ class SpoWebSetCommand extends SpoCommand {
         title: typeof args.options.title !== 'undefined',
         quickLaunchEnabled: args.options.quickLaunchEnabled,
         footerEnabled: args.options.footerEnabled,
+        navAudienceTargetingEnabled: args.options.navAudienceTargetingEnabled,
```

Comment:
Right now, if we don't specify this option, this option will be excluded from the telemetry. We are expecting a `true` if this option is specified (doesn't matter whether true or false is specified).
Therefore I suggest that we change this to use `typeof option !== 'undefined'`. Seems like the other boolean options do the same, could you fix these as well?